pytest_run.py

```diff
@@ -11,7 +11,7 @@
 
```

Comment:
Above this, in too-far-from-the-diff-to-make-suggestions territory, is a `coding=utf-8` line that should be removed.